lib/sourceParams.js

```diff
@@ -127,9 +127,14 @@ module.exports = function (app) {
     params.htmlValidator.enable = false
   }
   if (flags.backgroundValidator) {
-    params.htmlValidator.separateProcess.enable = true
+    params.htmlValidator.separateProcess = true
```

Comment:
`separateProcess` was made into an object when the autoKiller was added, so ensure you're interacting with the `enable` property. (This is why the build failed epicly)

Reply:
Gotcha, I have added the `enable` property back to the `separateProcess` object